### ISSUE TITLE
Segmentation fault on Windows

### ISSUE THREAD

**Issue by adament** [rpy2/rpy2#759]
As mentioned in #754 I get a segfault on Windows when trying to use current master on Windows.

**To Reproduce**

So far the only way I have been able to build the package has been using the recipe from [conda-forge](https://github.com/conda-forge/rpy2-feedstock) and then I get the segmentation fault when trying to import robjects.

However it looks like the current CI also segfaults on Windows https://github.com/rpy2/rpy2/runs/1606566615 which is probably a better place to start debugging.

I think I managed to isolate the first occurence of the segmentation fault to d763939.

**Comment by adament** [comment]
https://github.com/rpy2/rpy2/blob/bc91910994af7028d777c5854eed9918c53db2e0/rpy2/rinterface_lib/R_API.h#L378-L380

on the version of R 4.0 I am using this should be
```c
extern char *getDLLVersion(void);
extern char *getRUser(void);
extern char *get_R_HOME(void);
```
note the last 2 lines are not function pointers.

**Comment by lgautier** [comment]
> However it looks like the current CI also segfaults on Windows https://github.com/rpy2/rpy2/runs/1606566615 which is probably a better place to start debugging.

Yes, there is a segfault. The CI is here in case there is a contribution from the Windows community. This provides a target for what should ideally work in the end. Otherwise the source for the segfault might with the rpy2, or the way the environment for CI is configured.

I don't have the bandwidth to work much on the Windows version myself.

**Comment by lgautier** [comment]
The CI pipeline is now building binary wheels for Windows. Can someone check if the issue is still present?

The binary wheels can be fetched from CI pipeline runs on Github. Otherwise I have uploaded dev binary to pypi that can be installed with:

```sh
pip install -U --pre rpy2
```
